vector/src/main/java/im/vector/app/features/home/room/list/home/invites/InvitesViewModel.kt

```diff
@@ -129,8 +142,26 @@ class InvitesViewModel @AssistedInject constructor(
                 sortOrder = RoomSortOrder.ACTIVITY
         )
 
-        setState {
-            copy(pagedList = pagedList)
-        }
+        pagedList.asFlow()
+                .map {
+                    if (it.isEmpty()) {
```

Comment:
out of interest, is an empty result only returned after the `pagedList` has finished loading?

or is it possible for it to emit an empty state instantly?